pachyderm/values.yaml

```diff
@@ -1,116 +1,256 @@
+# imagePullSecret sets the image pull secret used for all images.  It
```

Comment:
Just out of curiosity, what organization methodology are you using in re-organizing this file?